cpptoml_config.h.in

```diff
@@ -0,0 +1,4 @@
+/************************** HEADER FILES *************************/
+
+/* Define to 1 if you use external cpptoml library. */
+#cmakedefine SPDLOG_SETUP_CPPTOML_EXTERNAL 1
```

Comment:
Nitpick, would prefer the file to look more like the rest of the files like this:

```cpp
#pragma once

// define to 1 if you use external cpptoml library
#cmakedefine SPDLOG_SETUP_CPPTOML_EXTERNAL 1
```

Also, would prefer to shift this file into the `cmake` directory since it doesn't really need to be exposed to the repo root directory. The `CMakeLists.txt` will need to make a slight change to the locate this file.